packages/discord.js/src/managers/ApplicationCommandPermissionsManager.js

```diff
@@ -218,17 +211,31 @@ class ApplicationCommandPermissionsManager extends BaseManager {
       }
     }
 
-    return this.set({ guild: guildId, command: commandId, permissions: newPermissions });
+    return this.set({ guild: guildId, command: commandId, permissions: newPermissions, token });
   }
 
+  /**
+   * A static snowflake that identifies the everyone role for application command permissions.
+   * It is the same as the guild id
+   * @typedef {Snowflake} RolePermissionConstant
+   */
+
+  /**
+   * A static snowflake that identifies the "all channels" entity for application command permissions.
+   * It will be the result of the calculation `guildId - 1`
+   * @typedef {Snowflake} ChannelPermissionConstant
+   */
+
   /**
    * Options used to remove permissions from a command
-   * <warn>The `command` parameter is not optional when the managers `commandId` is `null`</warn>
+   * <warn>Omitting the `command` parameter removes from the guild wide permissions
+   * when the managers `commandId` is `null`</warn>
+   * <warn>At least one of `users`, `roles`, and `channels` is required</warn>
    * @typedef {BaseApplicationCommandPermissionsOptions} RemoveApplicationCommandPermissionsOptions
-   * @property {UserResolvable|UserResolvable[]} [users] The user(s) to remove from the command permissions
-   * <warn>One of `users` or `roles` is required</warn>
-   * @property {RoleResolvable|RoleResolvable[]} [roles] The role(s) to remove from the command permissions
-   * <warn>One of `users` or `roles` is required</warn>
+   * @property {string} token The bearer token to use that authorizes the permission removal
+   * @property {UserResolvable[]} [users] The user(s) to remove
+   * @property {Array<RoleResolvable|RolePermissionConstant>} [roles] The role(s) to remove
```

Comment:
iirc Array<> isn't used in the docs
```suggestion
   * @property {(RoleResolvable|RolePermissionConstant)[]} [roles] The role(s) to remove
```

Reply:
well, it is (and its how its displayed anyways), and this cannot be parsed by jsdoc

Reply:
I’m pretty sure I’ve seen this be used in the docs before but aight

Reply:
causes this ci error: https://github.com/discordjs/discord.js/runs/6541661129?check_suite_focus=true